packages/core/src/features/css-type.ts

```diff
@@ -48,19 +48,32 @@ export const hooks = createFeature<{
     },
     transformSelectorNode({ context, node, selectorContext }): void {
         const resolvedSymbols = context.getResolvedSymbols(context.meta);
-        const resolved = resolvedSymbols.element[node.value] || [
-            // provides resolution for native elements
-            // that are not collected by parts
-            // or elements that are added by js mixin
-            {
-                _kind: 'css',
-                meta: context.meta,
-                symbol: { _kind: 'element', name: node.value },
-            },
-        ];
+        let resolved = resolvedSymbols.element[node.value];
+        if (!resolved) {
+            const resolvedClass = resolvedSymbols.class[node.value];
```

Comment:
want to do that, but keeping this to be done as the last change in this PR, in order to keep review simple

Reply:
We want to move it to core for 2 reasons:
1. to reuse in the formatter
2. to move a concern of alias type selector on an invalid node from the transformer to the analyzer

Both are less important atm, and I will move it in a future PR, when it will be requires.